rocksdb_admin/tests/application_db_test.cpp

```diff
@@ -0,0 +1,196 @@
+/// Copyright 2016 Pinterest Inc.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+/// http://www.apache.org/licenses/LICENSE-2.0
+
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+//
+// @kangnan li (kangnanli@pinterest.com)
+//
+
+#include <stdlib.h>
+#include <ctime>
+#include <iostream>
+#include <thread>
+#include <vector>
+
+#include "boost/filesystem.hpp"
+#include "gtest/gtest.h"
+#include "rocksdb/db.h"
+#include "rocksdb/merge_operator.h"
+#include "rocksdb/sst_file_writer.h"
+#include "rocksdb/status.h"
+#include "rocksdb_admin/application_db.h"
+#include "rocksdb_replicator/rocksdb_replicator.h"
+
+namespace admin {
+
+using boost::filesystem::remove_all;
+using rocksdb::AssociativeMergeOperator;
+using rocksdb::DB;
+using rocksdb::DestroyDB;
+using rocksdb::EnvOptions;
+using rocksdb::Logger;
+using rocksdb::Options;
+using rocksdb::Slice;
+using rocksdb::SstFileWriter;
+using rocksdb::Status;
+using std::list;
+using std::make_unique;
+using std::pair;
+using std::string;
+using std::to_string;
+using std::unique_ptr;
+using std::vector;
+
+class SimpleMergeOperator : public AssociativeMergeOperator {
```

Comment:
why is the MergeOperator needed in the test?

Reply:
Yes, it is not needed. Removed.